src/library_pthread.js

```diff
@@ -477,7 +477,8 @@ var LibraryPThread = {
 #if PTHREADS_DEBUG
       out('Allocating a new web worker from ' + pthreadMainJs);
 #endif
-      PThread.unusedWorkers.push(new Worker(pthreadMainJs));
+      var workerName = 'pthread #' + PThread.unusedWorkers.length;
+      PThread.unusedWorkers.push(new Worker(pthreadMainJs, {'name' : workerName}));
```

Comment:
Seems reasonable.

Its a little misleading because the workers get re-used so if you have 1 worker that gets used over and over it would still stay `'pthread #1` even if it was the 10th thread (assuming it was joined and returned to the pool each time).

I guess thats fine.  Perhaps it would be more clear to call them `pthread worker #N` ?  Or `pthread pool #N`?  I don't feel strongly.

Reply:
Interesting point @sbc100 

Yes, perhaps "pthread worker #N" would be slightly clearer.